packages/paste-website/src/pages/roadmap/index.mdx

```diff
@@ -0,0 +1,118 @@
+---
+
+import {Box} from '@twilio-paste/box';
+import {AnchoredHeading} from '../../components/Heading';
+import {Callout, CalloutTitle, CalloutText} from '../../components/callout';
+
+<content>
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+<p>{props.pageContext.frontmatter.description}</p>
+
+<Box marginTop={40}>
+
+<AnchoredHeading>Release 0.3</AnchoredHeading>
+
+<Callout>
+  <CalloutText>Codenamed <strong>Oh! Point Three!</strong> This release is expected to land in the <em>last week of October.</em></CalloutText>
+</Callout>
+
+### Component Updates
+
+#### Text &amp; Box Primitives
+We'll polish our Text and Box Primitives. Together, these primitive components will be used to build significant amounts of the Paste Design System. Also expect to see guides for how to use them to build your own interfaces in a future release.
+
+#### Implement a paste-core package
+We recognize that it can be tedious to install multiple commonly used components across every project you work on. In 0.3, we'll implement `paste-core`, a wrapper component that you can install for common components. Yay for quality of life!
+
+#### Heading, Paragraph Components
+We'll expand on the Text primitive to build out Heading and Paragraph components.
+
+#### Card Component
+We're designing and building a unified card component for Paste. This will replace our legacy card soup with a composable card that you can customize for your products.
+
+#### Grid Component (Design Preview)
+A standard 12-column grid for Paste will drop in 0.3. There's been a few discussions around responsiveness, and to give some clarity:
+
+- This will be a responsive grid layout system
+- All components will be designed to be 100% width to their containers.
+
+_Note:_ Just like accessibility, responsiveness is not an out-of-the-box solution. Consumers of any design system still need to compose their responsive screens based on their unique requirements. We'll provide the layout system and fluid components. Teamwork!
+
+#### Flex Component
+A layout component that simplifies aligning a single horizontal row of elements. Also has a fondness for protein shakes.
+
+#### Paste Utilities
+You remember the first time you uploaded a profile picture to AIM and it was skewed out? No, that was just us? Well, the Aspect Ratio utility will drop in this release and give guidance on how to maintain proportions while embedding images, video, media, and live components.
+
+Oh, we'll also ship an Absolute utility for positioning stuff on a page.
+
+#### Alert &amp; Toast Components (Design Preview)
+We're going to share designs and documentation for our proposal on unified Alert and Toast components. As always, these will be pure UI, and not concerned with any notification engines or product specific code.
+
+### Community &amp; Education Updates
```

Comment:
This heading doesn't have a hash link on it.

Reply:
I think it's the entity encoded ampersand thats causing the issue.

Try replacing all the `&amp;` with `&` for now

Reply:
I've made this change to fix the headers.